resources/aws_ec2.go

```diff
@@ -175,6 +186,24 @@ func (obj *AwsEc2Res) Init() error {
 	obj.closeChan = make(chan struct{})
 	obj.wg = &sync.WaitGroup{}
 
+	if obj.WatchListenAddr != "" {
+		// make sns client
+		snsSess, err := session.NewSession(&aws.Config{
+			Region: aws.String(obj.Region),
+		})
+		if err != nil {
+			return errwrap.Wrapf(err, "error creating sns session")
+		}
+		obj.snsClient = sns.New(snsSess)
+
+		// make the sns topic
+		topicArn, err := obj.snsMakeTopic()
+		if err != nil {
+			return errwrap.Wrapf(err, "error making sns topic")
+		}
+		obj.snsTopicArn = topicArn
```

Comment:
struct variant is called `obj.snsTopicArn` so why not call the var `snsTopicArn` ?